R/forest_plot.R

```diff
@@ -90,11 +90,11 @@ forest_plot <- function(df, facet_by = NA, shape_by = NA, col_by = NA,
     ## use the palette if provided, otherwise use the default
     ## as defined in epireview
     ## if neither is provided, use the default palette
-    if (!is.na(shp_palette)) {
+    if (!is.null(shp_palette)) {
       p <- p + scale_shape_manual(values = shp_palette, na.value = 4)
```

Comment:
@shaziaruybal Can you check that this na.value param is fine? I've added it based on the merge conflicts, but it wasn't in the initial version of develop that you were working on

Reply:
@tristan-myles and I have checked and we think the na.value param is fine